config/routes.rb

```diff
@@ -7,6 +7,8 @@
 
   resources :movies, only: [:index, :show]
   resources :texts, only: [:index, :show] do
-    resource :readed_texts, only: [:create, :destroy]
+  resource  :readed_texts, only: [:create, :destroy]
   end
+
+  resources :lines
```

Comment:
必要なルーティングのみにしておきましょう。

```suggestion
  resources :lines, only: [:index, :show]
```